src/lib/y2partitioner/clients/main.rb

```diff
@@ -1,17 +1,34 @@
+require "y2partitioner/widgets/overview"
+require "y2storage"
+
 Yast.import "CWM"
 Yast.import "Stage"
 Yast.import "Wizard"
 
 module Y2Partitioner
   module Clients
+    # Main entry point to see partitioner configuration
     class Main
       extend Yast::I18n
       extend Yast::UIShortcuts
 
       def self.run
         textdomain "storage"
 
-        contents = HBox(
+        staging = Y2Storage::StorageManager.instance.y2storage_staging
```

Comment:
because it is client for staging graph. So widget is independent, but client is make for some purpose ( edit staging ). if needed in future it can be changed, but I want to have at less code as possible in src/clients